File: 백준/Bronze/2480. 주사위 세개/주사위 세개.java
import java.util.*;
import java.lang.*;
import java.io.*;

class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String s = br.readLine();
        StringTokenizer st = new StringTokenizer(s);
        int[] nums = new int[3];
        for(int i=0; i<3; i++) nums[i] = Integer.parseInt(st.nextToken());

        Arrays.sort(nums);
        if (nums[0] == nums[1] && nums[1] == nums[2]) System.out.println(10000+nums[0]*1000);
        else if (nums[0] == nums[1]) System.out.println(1000+nums[0]*100);
        else if (nums[0] == nums[2]) System.out.println(1000+nums[0]*100);
        else if (nums[1] == nums[2]) System.out.println(1000+nums[1]*100);
        else  System.out.println(nums[2]*100);
    }
}

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.swift
import Foundation

var arr = readLine()!.split(separator: " ").map{Int(String($0))!}
arr.sort()

if (arr[0] == arr[1] && arr[1] == arr[2]) { print(10000 + arr[0]*1000) } 
else if (arr[0] == arr[1]) { print(1000 + arr[0]*100) } 
else if (arr[1] == arr[2]) { print(1000 + arr[1]*100) } 
else if (arr[0] == arr[2]) { print(1000 + arr[0]*100) } 
else { print(arr[2]*100) }